messages/create.sandbox.md

```diff
@@ -123,3 +123,19 @@ The poll interval (%d seconds) can't be larger that wait (%d in seconds)
 # error.noCloneSource
 
 Could not find the clone sandbox name "%s" in the target org.
+
+# error.userGroupQueryFailed
+
+Could not query user group.
+
+# error.bothApexClassIdAndNameProvided
+
+Could not specify both apexClassId and apexClassName.
+
+# error.bothUserGroupIdAndNameProvided
+
+Could not specify both userGroupId and userClassName.
```

Comment:
```suggestion
You cannot specify both `ActivationUserGroupId` and `ActivationUserGroupName` in the definition file at the same time.
```

Reply:
```suggestion
You can't specify both `ActivationUserGroupId` and `ActivationUserGroupName` in the definition file at the same time.
```

Reply:
Ditto, I have a small tweak to Cristian's suggestion.